module/send_results.py

```diff
@@ -5,9 +5,17 @@
 def send_results(rows: List, chat_id: int, context: CallbackContext) -> None:
     res = ""
     for i in range(len(rows)):
-        res += ("n°: " + str(i + 1) + "\n" + "ISBN: " + rows[i][1] + "\n" + "Titolo: " + rows[i][2] + "\n" + "Autori: " + rows[i][3] + "\n" + "Venditore: " + rows[i][4] + "\n" + "Prezzo: " + rows[i][5] + " €\n\n")
+        res += ("n°: " + str(i + 1) + "\n" + get_item_info(rows[i][1], rows[i][2], rows[i][3], rows[i][4], rows[i][5]) + "\n")
         if (i + 1) % 3 == 0:
             context.bot.send_message(chat_id, res)
             res = ""
     if res != "":
         context.bot.send_message(chat_id, res)
+
+def get_book_info(isbn: str, title: str, authors: str) -> str:
+    s = "ISBN: " + isbn + "\n" + "Titolo: " + title + "\n" + "Autori: " + authors + "\n"
```

Comment:
you can use the join or you can make it simpler using `f-string`
```suggestion
    s = f"ISBN: {isbn} \nTitolo: {title}\nAutori: {authors}\n"
```